README.md

```diff
@@ -67,14 +67,64 @@ or load from a CDN:
 };
 ```
 
+## Example with Header mapping
+
+```jsx
+// import jsonToCsvExport from "json-to-csv-export";
+  const mockData = [
+    {
+      id: 1,
+      firstName: "Sarajane",
+      lastName: "Wheatman",
+      email: "swheatman0@google.nl",
+      language: "Zulu",
+      ip: "40.98.252.240",
+    },
+    {
+      id: 2,
+      firstName: "Linell",
+      lastName: "Humpherston",
+      email: "lhumpherston1@google.com.br",
+      language: "Czech",
+      ip: "82.225.151.150",
+    },
+  ];
+
+  const headers = [
+    { key: "id", label: "Identifier" },
+    { key: "firstName", label: "First Name" },
+    { key: "lastName", label: "Last Name" },
+    { key: "email", label: "Email Address" },
+    { key: "language", label: "Language" },
+    { key: "ip", label: "IP Address" },
+  ];
+
+  return (
+    <button onClick={() => jsonToCsvExport({ data: mockData, headers })}>
+      Download Data
+    </button>
+  );
```

Comment:
This example isn't rendering correctly at runtime, needs to be wrapped in a function:
```suggestion
() => {
  const mockData = [
    {
      id: 1,
      firstName: "Sarajane",
      lastName: "Wheatman",
      email: "swheatman0@google.nl",
      language: "Zulu",
      ip: "40.98.252.240",
    },
    {
      id: 2,
      firstName: "Linell",
      lastName: "Humpherston",
      email: "lhumpherston1@google.com.br",
      language: "Czech",
      ip: "82.225.151.150",
    },
  ];

  const headers = [
    { key: "id", label: "Identifier" },
    { key: "firstName", label: "First Name" },
    { key: "lastName", label: "Last Name" },
    { key: "email", label: "Email Address" },
    { key: "language", label: "Language" },
    { key: "ip", label: "IP Address" },
  ];

  return (
    <button onClick={() => jsonToCsvExport({ data: mockData, headers })}>
      Download Data
    </button>
  );
  };
```
<img width="767" alt="Screenshot 2024-10-01 at 4 47 48 PM" src="https://github.com/user-attachments/assets/62b47820-5fd4-4b42-bc2c-4655cfa03688">